terraform/accounts/verify/clusters/prod/cluster.tf

```diff
@@ -0,0 +1,112 @@
+terraform {
+  backend "s3" {}
+}
+
+variable "aws_account_role_arn" {
+  type = "string"
+}
+
+provider "aws" {
+  region = "eu-west-2"
+  assume_role {
+    role_arn = "${var.aws_account_role_arn}"
+  }
+}
+
+data "aws_caller_identity" "current" {}
+
+module "gsp-cluster" {
+    source = "git::https://github.com/alphagov/gsp-terraform-ignition//modules/gsp-cluster"
+    cluster_name = "prod"
+    controller_count = 2
+    controller_instance_type = "m5d.large"
+    worker_count = 3
+    worker_instance_type = "m5d.large"
+    /* etcd_instance_type = "t3.medium" */
+    dns_zone = "verify.govsvc.uk"
+    user_data_bucket_name = "gds-verify-gsp-state"
+    user_data_bucket_region = "eu-west-2"
+    k8s_tag = "v1.12.2"
+    admin_role_arns = [
+      "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/admin",
+      "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/deployer",
+    ]
+    gds_external_cidrs = [
+      "213.86.153.212/32",
+      "213.86.153.213/32",
+      "213.86.153.214/32",
+      "213.86.153.235/32",
+      "213.86.153.236/32",
+      "213.86.153.237/32",
+      "85.133.67.244/32",
+      "18.130.144.30/32", # autom8 concourse
+    ]
+    addons = {
+      ingress = 1
+      canary = 0
+      monitoring = 1
+      secrets = 1
+      ci = 0
+    }
+}
+
+output "bootstrap-base-userdata-source" {
```

Comment:
I think the `output` stanzas should all live in a single file, separate to everything else. I'm sure over time we're going to need to add `output` entries to feed bootstrappers or upgraders and it will save a lot of hassle if we can copy a whole file around than just a few lines...I'd even suggest this file could even be a symlink to a common one if I trusted that to work!

Reply:
yep I mentioned this in the follow up ticket ... it became very clear when writing this pipeline that we cannot expect to keep them in sync